docs/products/postgresql/concepts/pg-connection-pooling.md

```diff
@@ -2,28 +2,38 @@
 title: Connection pooling
 ---
 
-Connection pooling in Aiven for PostgreSQL® services allows you to
-maintain very large numbers of connections to a database while
-minimizing the consumption of server resources.
+Connection pooling in Aiven for PostgreSQL® services allows you to maintain very large numbers of connections to a database while minimizing the consumption of server resources.
+
+## About connection pooling
 
 Aiven for PostgreSQL connection pooling uses
 [PgBouncer](https://www.pgbouncer.org/) to manage the database
 connection.
 
-:::note
-Each pool can handle from a minimum of 5000 client connections to a
-maximum defined by the lower threshold between:
-
--   500 for each GB of RAM in the service plan
--   A total of 50000 client connections
-:::
-
 Unlike when you connect directly to the PostgreSQL® server, each client
 connection does not require a separate backend process on the server.
 PgBouncer automatically inserts the client queries and only uses a
 limited number of actual backend connections, leading to lower resource
 usage on the server and better total performance.
 
+## Maximum number of client connections
+
+To check the maximum number of client connections for your service, take into account the
+number of the connection pools and RAM GBs your service uses, and apply the following rules:
+
+- Each pool can handle at least 5000 connections.
+- This number (5000) increases for a pool by 500 for each GB of RAM as per your service plan.
+- 50000 is the maximum number of client connections per service.
+
+For example, if a service using Business-4 plan with 4 GB of RAM has two connection pools,
```

Comment:
@0xlianhu I would appreciate if you could validate this example. Thanks.

Reply:
We cannot release this document until we fix the bug I mentioned in the slack channel.

Reply:
According to others' discussion and existing test cases, I think the document should be updated to:
minimum 5000, max 50000. 

Hence,  for a 4G machine, it should be 5000 instead of 7000. 

I.e., 
        (1024 * 2, 5000),
        (1024 * 4, 5000),
        (1024 * 8, 5000),
        (1024 * 16, 8000),
        (1024 * 32, 16000),
        (1024 * 64, 32000),
        (1024 * 128, 50000),
        (1024 * 256, 50000),